src/Files.App/UserControls/SidebarControl.xaml.cs

```diff
@@ -250,34 +241,6 @@ private List<ContextMenuFlyoutItemViewModel> GetLocationItemMenuItems(INavigatio
 					ShowItem = options.IsLocationItem
 				},
 				new ContextMenuFlyoutItemViewModel()
-				{
-					Text = "SideBarFavoritesMoveToTop".GetLocalizedResource(),
-					Glyph = "\uE11C",
-					Command = MoveItemToTopCommand,
-					ShowItem = showMoveItemUp
-				},
-				new ContextMenuFlyoutItemViewModel()
-				{
-					Text = "SideBarFavoritesMoveOneUp".GetLocalizedResource(),
-					Glyph = "\uE70E",
-					Command = MoveItemUpCommand,
-					ShowItem = showMoveItemUp
-				},
-				new ContextMenuFlyoutItemViewModel()
-				{
-					Text = "SideBarFavoritesMoveOneDown".GetLocalizedResource(),
-					Glyph = "\uE70D",
-					Command = MoveItemDownCommand,
-					ShowItem = showMoveItemDown
-				},
-				new ContextMenuFlyoutItemViewModel()
-				{
-					Text = "SideBarFavoritesMoveToBottom".GetLocalizedResource(),
```

Comment:
Can you remove the strings as well?

Reply:
I will try to reimplement this, this is a temporary removal

Reply:
It's alright if it's not done in this pull request

Reply:
Ok

Reply:
I'll open another PR when I find a reliable way of doing this